src/services/object-diff.ts

```diff
@@ -0,0 +1,99 @@
+import { isString } from './common';
```

Comment:
I would probably call this `aws-diff.ts` since we will eventually add more cloud providers and this is only a thing for AWS